ConsoleApp1/ServerRoom.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ConsoleApp1
+{
+   
+        class Showall
+        {
+            Server NewServer = new Server();
+            public void show()
+            {
+                for (int x = 0; x < NewServer.checker; x++)
+                {
+                    Console.WriteLine(NewServer.name[x]);
+                    Console.WriteLine(NewServer.middleinitial[x]);
+                    Console.WriteLine(NewServer.yearlevel[x]);
+                    Console.WriteLine(NewServer.course[x]);
+                }
+            }
+        }
+
+        public class Account
+        {
+            Server NewServer = new Server();
+            public void newaccount()
+            {
+                Console.WriteLine("\n\nname:");
+                string name = Console.ReadLine();
+                Console.WriteLine("middle initial:");
+                char minitial = Convert.ToChar(Console.ReadLine());
+                Console.WriteLine("year level:");
+                int yl = Convert.ToInt32(Console.ReadLine());
+                Console.WriteLine("course:");
+                string course = Console.ReadLine();
+
+                NewServer.setname(name);
+                NewServer.setmiddleinitial(minitial);
+                NewServer.setyearlevel(yl);
+                NewServer.setcourse(course);
+                NewServer.checkerplusone();
```

Comment:
Create a model for this:

`public class Student{
public string Name {get;set;}
public string MiddleInitial {get;set;}
public int YearLevel {get;set;}
public string Course {get;set;}
}`